include/evmc/helpers.h

```diff
@@ -19,26 +19,39 @@
 
 #include <evmc/evmc.h>
 
+#define EVMC_INLINE
+#ifdef __cplusplus
+#  undef EVMC_INLINE
+#  define EVMC_INLINE inline
+#endif
+#if __STDC__
+#  if __STDC_VERSION__ >= 199901L
```

Comment:
Could also merge these into `if defined(__cplusplus) || __STD_VERSION__ >= 199901L` if we assume the preprocessor for C90-only compiler is clever enough...

Reply:
This would work, because if `__STDC_VERSION__` is undefined it will be substituted with `0`.

Reply:
By the clever enough I meant that it supports the `defined()` expression as well as the `or` operator`.